pallets/stellar-relay/src/testing_utils.rs

```diff
@@ -197,7 +197,7 @@ pub fn build_dummy_proof_for<T: crate::Config>(
 	txes.push(transaction_envelope.clone()).unwrap();
 	let transaction_set = TransactionSet { previous_ledger_hash: Hash::default(), txes };
 
-	let tx_set_hash = crate::compute_non_generic_tx_set_content_hash(&transaction_set);
```

Comment:
Don't we need to handle this now? You changed it to return an `Option` now, but the `create_scp_envelope` function that consumes it a few lines below still expects the real type and not an option I think.

Reply:
I added `unwrap` method but now i replace it with 
`.expect("Should compute non generic tx set content hash");`